kani-compiler/src/args.rs

```diff
@@ -47,6 +47,9 @@ pub struct Arguments {
     pub reachability_analysis: ReachabilityType,
     #[clap(long = "enable-stubbing")]
     pub stubbing_enabled: bool,
+    /// Option name used to tell the compiler to execute the list subcommand
```

Comment:
Per slack, no -- will fix. Going to move to draft until then, since this will take me a bit of time to do and I don't want anyone wasting time reviewing in the meantime.